src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -254,13 +254,14 @@ def privacy_request_csv_download(
 
     csv_file.writerow(
         [
-            "Time received",
-            "Subject identity",
-            "Policy key",
-            "Request status",
-            "Reviewer",
-            "Time approved/denied",
-            "Denial reason",
+            "Status",
+            "Request Type",
+            "Subject Identity",
+            "Time Received",
+            "Reviewed By",
+            "Request ID",
+            "Time Approved/Denied",
+            "Denial Reason",
```

Comment:
What drove the re-ordering here? Might it be better to start each row with `Request ID`?

Reply:
The reordering was done to make the columns match the table in the UI. I'm pretty sure that's what the ticket was asking for

> We should update this to be consistent with the Admin UI and include the following columns in the download:

>Status (e.g. New, Error, Pending...)
Request Type (e.g. Access, Erasure)
Subject Identity (e.g. email address)
Time Received (e.g. January 25, 2023...)
Reviewed By
Request ID (e.g. pri_8bfe3955-cbab-47da-b106-fe5d4361e8dd)
Time Approved/Denied
Denial Reason


Since it said to make in consistent with the Admin UI and listed the columns in that order I went ahead an reordered them. That does makes sense though. We may want to see what product thinks about that. I can make that change real quick but I think they'll want to sign off on moving the `Request ID` to the first column

Reply:
Yeah, reading this ticket I think @TheAndrewJackson is right: https://github.com/orgs/ethyca/projects/18/views/6?pane=issue&itemId=18915982
I'd order to match the UI